src/aks-preview/azext_aks_preview/addonconfiguration.py

```diff
@@ -11,6 +11,11 @@
 from azure.cli.core.commands.client_factory import get_subscription_id
 from azure.cli.core.util import sdk_no_wait, send_raw_request
 from azure.core.exceptions import HttpResponseError
+from azure.cli.command_modules.acs.addonconfiguration import (
+    ensure_container_insights_for_monitoring,
+    sanitize_loganalytics_ws_resource_id,
+    ensure_default_log_analytics_workspace_for_monitoring
```

Comment:
Seems the function is redefined on line 325?

Reply:
> Seems the function is redefined on line 325?

> Yes, thanks for catching this. fixed.